Assets/Scripts/Game/Player.cs

```diff
@@ -31,7 +32,7 @@ public class Player : Entity
 
     private PositionChangedEvent onPositionChanged = new PositionChangedEvent();
 
-    void Awake()
+    protected override void Awake()
```

Comment:
Do you not need to call `base.Awake()` as well?

Reply:
Good catch!